src/components/widgets/TextWidget/TextWidget.tsx

```diff
@@ -0,0 +1,33 @@
+import React from 'react'
+import {WidgetTextMeta} from 'interfaces/widget'
+import {connect} from 'react-redux'
+import {Store} from '../../../interfaces/store'
+import styles from './TextWidget.less'
+
+const parse = require('html-react-parser')
```

Comment:
It is preferrable to use `import` syntax instead of `require`.
`import` syntax is the official JS module system introduced in [ES2015](https://www.ecma-international.org/ecma-262/6.0/#sec-imports).
`require` syntax is unofficial JS module system called "CommonJS" used in NodeJS environment, which nowadays also  has a support for `import` syntax.

Additionaly, you'll need typins for 'marked' library as it is distributed without them:
```sh
yarn add @types/marked -D
```

```tsx
import parse from 'html-react-parser'
import marked from 'marked'
```

npm install @types/marked